### DISCUSSION TITLE
How should I update Penumbra?

### DISCUSSION THREAD

**Discussion by dovochi** [xivdev/Penumbra#55] [Q&A] [answered]
I'm still using v0.3.4.3 and it doesn't work anymore with the new Dalamud release. I noticed that Penumbra v0.4.4.5 is available now. Can I just replace the whole folder in Roaming or are there any other steps that I should do first? I've got a lot of mods installed.

**Comment by AncientAzem** [comment, marked as answer]
@dovochi I would reccomend removing the plugin from the `devPlugins` folder entirely and using the custom plugin repository going forward. The URL for the custom repo is https://raw.githubusercontent.com/xivdev/Penumbra/master/repo.json

**Reply by FiraliaDev** [reply]
Might be a stupid question, but how do I do that exactly?

Penumbra hasn't been able to apply any mods in-game for me since Dalamud did that plugins update a while ago. I've updated it to the latest release, but still nothing. 
I've had it in the `devPlugins` folder, could that be why?

**Reply by Ottermandias** [reply]
Yes. Please remove all remnants of penumbra from DevPlugins, then go to Dalamud settings (click in main menu or /xlsettings), to the Experimental tab.
There, add the above url (https://raw.githubusercontent.com/xivdev/Penumbra/master/repo.json) to Custom Plugin Repositories, click on the + button, enable the repository (verify the checkmark) and then click on Save.
Then you should be able to install penumbra via /xlplugins, and it would be kept up-to-date automatically.
